x-pack/plugins/cases/server/routes/api/cases/push_case.ts

```diff
@@ -38,10 +38,13 @@ export function initPushCaseApi({ router, logger }: RouteDeps) {
           fold(throwErrors(Boom.badRequest), identity)
         );
 
+        const url = getCaseUrl(request.headers.referer, params.case_id);
```

Comment:
@stephmilovic I think we need to think more about how to get the url from the server side, look at this [thread in slack](https://elastic.slack.com/archives/C5TQ33ND8/p1623836716186800). I think our idea was too simplistic and we might have more problem than good.

Reply:
what do you mean by "path the url" @XavierM? do you mean pass the url from the frontend? did you see @smith regex idea?

Reply:
Yes passing the url from the frontend and we will be finer.

Reply:
> what do you mean by "path the url" @XavierM? do you mean pass the url from the frontend? did you see @smith regex idea?

The problem is not with how we parse the url afterwards but how we get it in the first place. `request.headers.referer` is very error prone. It does not work when Kibana is behind a proxy. You can read more to the tread @XavierM posted.

Reply:
already outdated my dude @cnasikas ;P